packages/fxa-admin-server/src/subscriptions/stripe.service.ts

```diff
@@ -56,6 +56,10 @@ export class StripePaymentConfigManagerService extends PaymentConfigManager {
 
     super(config, firestore, logger);
   }
+
+  async onModuleDestroy() {
+    await this.stopListeners();
```

Comment:
This was important to prevent an error from happening during teardown. The error would complain about a bulkwrite operation not being closed.